src/types/table.ts

```diff
@@ -50,3 +63,11 @@ export interface TableStatePayload {
   state: TableState
   hasData: boolean
 }
+
+export interface TableRowAttributes {
+  [key: string]: string | RouteLocationRaw | undefined
```

Comment:
issues:

1. Since you're binding the `to` property below specifically for row links, you don't need to also include the `RouteLocationRaw` here for the catch-all property type
2. undefined syntax
```suggestion
  [key: string]?: string
```

Reply:
This is obsolete now